.github/workflows/pr-wasm-preview.yml

```diff
@@ -0,0 +1,221 @@
+name: PR WASM Preview Deployment
+
+on:
+  pull_request:
+    branches:
+      - main
+      - dev
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
```

Comment:
`unlabeled` shouldn't be needed if we used
```
if: contains(github.event.pull_request.labels.*.name, 'deploy: wasm-playground')
```
As suggested in the previous comment.

Reply:
done